src/libponyrt/sched/start.c

```diff
@@ -93,6 +96,7 @@ static int parse_opts(int argc, char** argv, options_t* opt)
     {
       case OPT_THREADS: opt->threads = atoi(s.arg_val); break;
       case OPT_MINTHREADS: opt->min_threads = atoi(s.arg_val); break;
```

Comment:
If I'm not mistaken this silently overflows… (e.g. if `-1` negative values are provided)